CSS/styles.css

```diff
@@ -0,0 +1,778 @@
+  background-color: whitesmoke;
+  border: 2px solid white;
+  padding: 0.7rem;
+  margin: 0.7rem 0;
+  width: 650px;
+  max-width: 100%;
+}
+
+.headline h2 {
+  font-size: 1.3rem;
+  font-weight: 900;
+  line-height: 2.2rem;
+  color: #5b5b5b;
+}
+
+p.venu {
+  font-size: 1rem;
+  font-weight: 400;
+  color: #5b5b5b;
+}
+
+#main-program {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: url(../Images/Backgrounds/program-bg.png);
+  background-color: #272a31;
+  background-size: 1.6rem;
+  padding: 2rem 0.8rem;
+  width: 100%;
+  gap: 1rem;
+}
+
+.featured-speakers h2 {
+  font-size: 1.2rem;
+}
+
+.main-program h2 {
+  font-size: 1.2rem;
+  color: #d3d3d3;
+}
+
+hr {
+  background-color: #ec5242;
+  border: 0.2px solid #ec5242;
+  border-radius: 2px;
+  width: 35px;
+}
+
+.program-info {
+  display: flex;
+  flex-direction: column;
+  margin: 1rem 0;
+  gap: 0.5rem;
+}
+
+.program-info li {
+  display: flex;
+  align-items: center;
+  background-color: rgba(255, 255, 255, 0.1);
+  transition: all 400ms linear 0.1ms;
+  padding: 1rem 0.5rem;
+  min-height: 5.5rem;
+  gap: 0.6rem;
+}
+
+.program-info img {
+  width: 2.5rem;
+}
+
+.program-info h3 {
+  color: #ec5242;
+  flex: 2;
+}
+
+.program-info p {
+  align-items: center;
+  font-size: 0.875rem;
+  line-height: 1.2rem;
+  color: #d3d3d3;
+  flex: 4;
+}
+
+#join-btn {
+  font-size: 1rem;
+  color: #fff;
+  background-color: #ec5242;
+  transition: all 400ms linear 0.1ms;
+  padding: 1.5rem;
+  cursor: pointer;
+  border: none;
+}
+
+.featured-speakers {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 2rem 0.8rem 3rem;
+  gap: 1rem;
+}
+
+.more-speakers {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #fff;
+  border: 1px solid #5a5a5a;
+  border-radius: 3px;
+  padding: 0.7rem 0;
+  margin-top: 1rem;
+  width: 100%;
+  gap: 0.5rem;
+}
+
+.fa-chevron-down {
+  font-size: 1rem;
+  color: #ec5242;
+}
+
+@media screen and (min-width: 768px) {
+  .hamburger,
+  #join-btn,
+  .more-speakers {
+    display: none;
+  }
+
+  .top-nav,
+  .logo,
+  .campaign,
+  .whole-program-btn,
+  #sponsor,
+  footer {
+    display: block;
+  }
+
+  .top-nav ul {
+    display: flex;
+    justify-content: right;
+    align-items: center;
+    background-color: #3e3e3e;
+    padding: 0 11%;
+    gap: 1.5rem;
+  }
+
+  .fa-brands {
+    color: #d3d3d3;
+  }
+
+  .top-nav a {
+    font-size: 0.8rem;
+    text-decoration: none;
+    line-height: 1.6rem;
+    color: #d3d3d3;
+  }
+
+  .bottom-nav {
+    padding: 0 11%;
+  }
+
+  .logo {
+    width: 10rem;
+    flex: 2;
+  }
+
+  .nav-menu {
+    left: auto;
+    width: auto;
+    height: auto;
+    position: initial;
+    flex-direction: row;
+    background: none;
+    padding: 0;
+    flex: 3;
+  }
+
+  .nav-menu li {
+    border: none;
+    width: auto;
+  }
+
+  a.nav-item {
+    font-size: 1rem;
+    line-height: normal;
+    color: #5a5a5a;
+  }
```

Comment:
- [ ] Great job 💪 . Kindly give your text the color of `#ec5242` as seen in the original design 🙏 

Original Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/205349164-8497b57c-d2f6-46e7-aa17-310ae80cda0a.png)  |  ![image](https://user-images.githubusercontent.com/74814780/205349195-f3814bf4-50db-4e74-ab7d-59224890d1b2.png)